.azure-devops/test/jobs/sanity.openjdk.yml

```diff
@@ -0,0 +1,44 @@
+parameters:
```

Comment:
i feel like there is now significant difference between this yaml file and sanity.functional.yml file, maybe we can merge those 2?

Reply:
I think you can take a look at our juniper-tests, we will pass a configurations object into the YAML templates and it will dynamically generate create those template.